Mage.Sets/src/mage/sets/MysteryBooster.java

```diff
@@ -44,3487 +41,1918 @@ public static MysteryBooster getInstance() {
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Acrobatic Maneuver", "KLD", "1"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Affa Protector", "OGW", "14"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Ainok Bond-Kin", "KTK", "3"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Alley Evasion", "AER", "6"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Angelic Purge", "SOI", "3"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Angelsong", "DDC", "15"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Apostle's Blessing", "MM2", "8"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Arrester's Zeal", "RNA", "4"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Artful Maneuver", "DTK", "4"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Aura of Silence", "C15", "60"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Bartered Cow", "ELD", "6"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Bonds of Faith", "DDQ", "2"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Borrowed Grace", "EMN", "14"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Bulwark Giant", "WAR", "7"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Caravan Escort", "DDP", "3"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Caught in the Brights", "AER", "10"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Celestial Crusader", "C14", "68"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Celestial Flare", "ORI", "8"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Center Soul", "DTK", "8"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Cliffside Lookout", "BFZ", "20"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Conviction", "AER", "12"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Countless Gears Renegade", "AER", "13"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Court Street Denizen", "DDO", "5"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Crib Swap", "C18", "65"));
+        boosterList.add(CardRepository.instance.findCardWithPreferredSetAndNumber("Danitha Capashen, Paragon", "DOM", "12"));
```

Comment:
Must use method like `addCard(name, set, number)` and check result of `findCardWithPreferredSetAndNumber` -- must be same set and number (if not then raise error). Current code is false positive on non existing cards.

Reply:
Fair point, I'll adjust this.